backend/webapi/Features/ClientLogs/ClientLogsController.cs

```diff
@@ -0,0 +1,16 @@
+namespace Pidp.Features.ClientLogs;
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+[Route("api/[controller]")]
+public class ClientLogsController
```

Comment:
All of our other controllers inherit from `PidpControllerBase`. This controller doesn't need to do that, necessarily, but it does need to at least have the `[ApiController]` annotation on it. Among other things, this annotation wires up the Fluent Validators, meaning currently there is no validation running against the `Create.Command`.